src/corelibs/U2Formats/src/ClustalWAlnFormat.h

```diff
@@ -34,21 +34,20 @@ class IOAdapter;
 class U2FORMATS_EXPORT ClustalWAlnFormat : public TextDocumentFormat {
     Q_OBJECT
 public:
-    static const QByteArray CLUSTAL_HEADER;
+    static const QString CLUSTAL_HEADER;
 
     ClustalWAlnFormat(QObject *p);
 
-    virtual void storeDocument(Document *d, IOAdapter *io, U2OpStatus &os);
+    void storeTextDocument(TextStreamWriter &writer, Document *d, U2OpStatus &os) override;
 
-    virtual void storeEntry(IOAdapter *io, const QMap<GObjectType, QList<GObject *>> &objectsMap, U2OpStatus &os);
+    void storeTextEntry(TextStreamWriter &writer, const QMap<GObjectType, QList<GObject *>> &objectsMap, U2OpStatus &os) override;
 
-protected:
-    virtual FormatCheckResult checkRawTextData(const QByteArray &rawData, const GUrl & = GUrl()) const;
+    FormatCheckResult checkRawTextData(const QString &dataPrefix, const GUrl & = GUrl()) const override;
 
-    virtual Document *loadTextDocument(IOAdapter *io, const U2DbiRef &dbiRef, const QVariantMap &fs, U2OpStatus &os);
+    Document *loadTextDocument(TextStreamReader &reader, const U2DbiRef &dbiRef, const QVariantMap &fs, U2OpStatus &os) override;
 
```

Comment:
Where is `DNASequence *loadTextSequence(TextStreamReader &reader, U2OpStatus &os) override`?

Reply:
ClustalW does not support DNA sequence. The default impl that sets error flag is enough